src/sagemaker_containers/_mpi.py

```diff
@@ -174,6 +174,11 @@ def _create_command(self):  # type: () -> List[str, Any]
 
         return command
 
+    def _python_executable(self):
```

Comment:
Maybe the name ```_python_executable``` implies that this function returns the location of the Python executable? I honestly don't have a better name suggestion though.

Reply:
that's a fair point. how about `_python_command`?

Reply:
sounds good!